test/rules/disallow-keywords-in-comments.js

```diff
@@ -0,0 +1,49 @@
+var Checker = require('../../lib/checker');
+var assert = require('assert');
+
+describe('rules/disallow-keywords-in-comments', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+        checker.configure({ disallowKeywordsInComments: true });
```

Comment:
Do one `describe` per option value. Right now you are only testing `true`. Likely why coverage is listed as going down.